src/components/Form.js

```diff
@@ -1,11 +1,57 @@
-import React from 'react';
-
-const Form = () => (
-  <form>
-    <input type="text" placeholder="Book title" />
-    <input type="text" placeholder="Book author" />
-    <button type="button">Add Book</button>
-  </form>
-);
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+import '../App.css';
+
+function Form() {
+  const [bookTitle, setBookTitle] = useState('');
+  const [bookAuthor, setBookAuthor] = useState('');
+  const dispatch = useDispatch();
+  const handleInputTitle = (bks) => {
+    setBookTitle(bks.target.value);
+  };
+
+  const handleInputAuthor = (bks) => {
+    setBookAuthor(bks.target.value);
+  };
+
+  const checkValidity = () => {
+    if (!bookTitle) return;
+    const bookObj = {
+      title: bookTitle,
+      author: bookAuthor,
+    };
+    dispatch(addBook(bookObj));
+    setBookTitle('');
+    setBookAuthor('');
+  };
```

Comment:
- [ ] Good job so far, I think it would be better if you check also the validity of the book author's input. It checks only for book title input when submitting but for the book author's input user can submit even if the input is empty. It can be better if the user can't submit the form if the author input is empty as well 🙃 🙃

Reply:
Thanks for pointing that out. I have fixed it.